packages/notifi-react-card/lib/utils/AlertConfiguration.ts

```diff
@@ -22,6 +22,9 @@ export const chatConfiguration = (): AlertConfiguration => {
       alertFrequency: 'THREE_MINUTES',
     },
     sourceType: 'NOTIFI_CHAT',
+    createSource: {
+      address: '*',
+    }
```

Comment:
Nit: Can you please add a comma at the end of createSource object?

```
createSource: {
      address: '*',
},
```
I received an eslint error alert so if you should check if eslint is working for you. :)